src/sst/core/configGraph.cc

```diff
@@ -987,8 +987,8 @@ ConfigGraph::getCollapsedPartitionGraph()
             for ( LinkId_t id : comp->allLinks() ) {
                 const ConfigLink& link = links[id];
 
-                if ( group.find(COMPONENT_ID_MASK(link.component[0])) == group.end() ||
-                     group.find(COMPONENT_ID_MASK(link.component[1])) == group.end() ) {
+                if ( group.find(COMPONENT_ID_MASK(link.component[0])) == group.end()
```

Comment:
I would do this change in a different PR and make this only CMake.

Reply:
@feldergast your call on this one.